x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/get_prebuilt_rules_and_timelines_status.ts

```diff
@@ -27,5 +30,7 @@ export const getPrebuiltRulesAndTimelinesStatus = async (
     .send()
     .expect(200);
 
+  await es.indices.clearCache({ index: ALL_SAVED_OBJECT_INDICES });
```

Comment:
I'd suggest to call this before calling the endpoint:

```ts
await es.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
await es.indices.clearCache({ index: ALL_SAVED_OBJECT_INDICES });

const response = await supertest...
```

Same in `getPrebuiltRulesStatus`.